test-network/hsm/softhsm2.conf

```diff
@@ -0,0 +1,7 @@
+# SoftHSM v2 configuration file
```

Comment:
I think putting this in the test-network directory would just cause more confusion, given that this is used by the application and not the infrastructure.

This can be moved to the app directory, or users can even create this file themselves given how small it is.